ivy_models/helpers/weights_helpers.py

```diff
@@ -29,7 +29,7 @@ def _map_weights(raw, ref, custom_mapping=None):
 def load_torch_weights(
     url,
     ref_model,
-    raw_keys_to_prune=[],
+    raw_keys_to_prune=["num_batches_tracked"],
```

Comment:
please keep the default value as `[]` and configure this in the unet `load_torch_weights` function